inlong-dashboard/src/ui/components/Layout/Tenant/index.tsx

```diff
@@ -106,24 +94,29 @@ const Comp: React.FC = () => {
   };
 
   const onFilter = allValues => {
```

Comment:
`allValues` has a plural meaning, but here value has only one parameter, it is recommended to change the parameter name to `keyword`.